spec/models/raw_input_changes_spec.rb

```diff
@@ -150,7 +150,7 @@
             organization_name: 'Best Gallery',
             phone_number: '1-800-123-4567',
             tag_names: 'invalid',
-            website: 'examplecom'
+            website: 'example.com'
```

Comment:
With this change, an invalid URL will blow up the whole row before we even attempt any of the other parts of applying the RawInput. If this is a problem, we'd have to refactor even more about how this class works, so I opted for the easy path.